ops/terraform/remote_files/scripts/start-bacalhau.sh

```diff
@@ -43,7 +43,8 @@ if [[ "${BACALHAU_NODE_NETWORK_TYPE}" == "nats" ]]; then
     --web-ui-port 80 \
     --labels owner=bacalhau \
     --requester-job-translation-enabled \
-    --default-publisher ipfs
+    --default-publisher local \
+    --local-publisher-address "${BACALHAU_LOCAL_PUBLISHER_ADDRESS}"
```

Comment:
nit: redundant to pass the cli flag as `BACALHAU_LOCAL_PUBLISHER_ADDRESS` env variable translates to the same config

Reply:
I don't think this flag actually expects an IP address, it expects one of these values: https://github.com/bacalhau-project/bacalhau/blob/main/pkg/lib/network/network.go#L111

Reply:
No it can be an IP address as well.  The problem here is that 'public' won't resolve to anything for google cloud as it doesn't give out public addresses to the VMs at runtime.

Reply:
Have pushed a code change, will redeploy to staging this evening.